internal/v5/auth.go

```diff
@@ -19,8 +21,10 @@ import (
 )
 
 const (
-	basicRealm        = "CharmStore4"
-	PromulgatorsGroup = "charmers"
+	PromulgatorsGroup      = "charmers"
+	opAccessCharmWithTerms = "access-operation"
```

Comment:
Let's add some comments here saying what these operations signify, please.

Also, how about using shorter names, as these will end up in cookies, and space is
at a premium there? e.g. op-access, op-other ?